Sources/SmithyCodegenCore/Shape/EnumShape.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import enum Smithy.Node
+import struct Smithy.ShapeID
+import enum Smithy.ShapeType
```

Comment:
Here & below are specialized Shape types for use in the Model.

All shapes descend from a `Shape` base class, similar to in Smithy codegen.